packages/experiments-realm/blog-app.gts

```diff
@@ -303,17 +279,80 @@ class BlogAppTemplate extends Component<typeof BlogApp> {
         font: 600 var(--boxel-font-lg);
         letter-spacing: var(--boxel-lsp-xxs);
       }
+      :deep(.card-view.categories-grid) {
+        --grid-card-height: 150px;
+      }
     </style>
   </template>
 
-  filters: LayoutFilter[] = new TrackedArray(this.args.model.filters);
-
   @tracked private selectedView: ViewOption = 'card';
-  @tracked private activeFilter: LayoutFilter = this.filters[0];
+  @tracked private activeFilter: LayoutFilter;
+  @tracked private filters: LayoutFilter[] = [];
 
   constructor(owner: Owner, args: any) {
     super(owner, args);
-    this.loadCardTypes.perform();
+    this.setFilters();
+    this.activeFilter = this.filters[0];
+  }
+
+  private get gridClass() {
+    if (this.activeFilter.displayName === 'Blog Posts') {
+      return 'blog-posts-grid';
+    } else if (this.activeFilter.displayName === 'Author Bios') {
+      return 'author-bios-grid';
+    } else if (this.activeFilter.displayName === 'Categories') {
+      return 'categories-grid';
+    }
+    return '';
+  }
+
+  private setFilters() {
+    let blogId = this.args.model.id;
+
+    let makeQuery = (codeRef: ResolvedCodeRef) => {
+      if (!blogId) {
+        throw new Error('Missing blog id');
+      }
+
+      return {
+        filter: {
+          on: codeRef,
+          eq: { 'blog.id': blogId },
```

Comment:
Previously, this filter was searching by relative urls, but that triggers the WAF error in staging/prod, because it doesn't like relative paths in url queries (it detects it as a security threat). 

So I changed it to searching by absolute urls, but that will only start working when we land https://github.com/cardstack/boxel/pull/1955

Reply:
The fix has landed

Reply:
🚀